acceptance-tests/src/test/java/it/com/atlassian/bitbucket/jenkins/internal/SmokeTest.java

```diff
@@ -333,7 +439,36 @@ private void runFullBuildFlow(BitbucketScmWorkflowJob workflowJob) throws IOExce
 
         // Verify BB has received build status
         fetchBuildStatusesFromBitbucket(commit.getId().getName()).forEach(status ->
-                assertThat(status, successfulBuildWithKey(workflowJob.getLastBuild().job.name)));
+                assertThat(status, successfulBuildWithKey(getBuildKey(workflowJob))));
+    }
+
+    private Response getBuildOperations(String commitId) {
+        return RestAssured
+                .given()
+                .log().ifValidationFails(LogDetail.ALL)
+                .auth().preemptive().basic(BITBUCKET_ADMIN_USERNAME, BITBUCKET_ADMIN_PASSWORD)
+                .header("Accept", JSON.getAcceptHeader())
+                .expect()
+                .statusCode(200)
+                .when()
+                .get(getInternalCommitUrl(forkRepo, commitId) + "/build-operations?key=" + getBuildKey(job));
+    }
+
+    private Response performAction(String commitId) {
+        JSONObject body = new JSONObject();
+        body.put("id", "ScheduleBuildAction");
+        body.put("key", getBuildKey(job));
+        return RestAssured
+                .given()
+                .log().ifValidationFails(LogDetail.ALL)
+                .auth().preemptive().basic(BITBUCKET_ADMIN_USERNAME, BITBUCKET_ADMIN_PASSWORD)
+                .header("Accept", JSON.getAcceptHeader())
+                .contentType("application/json")
+                .body(body)
+                .expect()
+                .statusCode(200)
+                .when()
+                .post(getInternalCommitUrl(forkRepo, commitId) + "/build-operations");
```

Comment:
I noticed "running actions from bitbucket server" in the title of this pull request and thought that perhaps Bitbucket Server now has a feature similar to `actions` in the [Checks API of GitHub](https://docs.github.com/en/rest/reference/checks) and [`ChecksAction` in the Checks API plugin for Jenkins](https://javadoc.jenkins.io/plugin/checks-api/io/jenkins/plugins/checks/api/ChecksAction.html), so that it could be used in <https://github.com/jenkinsci/checks-api-plugin/issues/65>. Alas, this test apparently uses a REST resource that is intended for the JavaScript UI of Bitbucket Server itself and cannot define any new build operations.